collector/collector.go

```diff
@@ -77,7 +87,7 @@ func getMetricURL(httpClient *http.Client, URL string, response interface{}) err
 	if err != nil {
 		return err
 	}
-
+	Tracef("Retrieved metric result:\n%s\n", string(body))
```

Comment:
make %q and drop string cast.

Reply:
Not sure if that's best.  We'll always be importing prettyfied JSON from NATS...   this would make it harder to read.  It could help w/ corrupted data through.

Here's %q, no cast:
e.g. 
```[11334] 2019/08/18 17:07:59.807111 [TRC] Retrieved metric result:
"{\n  \"start_time\": 1566152463,\n  \"current_time\": 1566169679,\n  \"uptime\": \"4h46m55.732814887s\",\n  \"request_count\": 0,\n  \"connectors\": [\n    {\n      \"name\": \"Stan:foo to Stan:foo.out\",\n      \"id\": \"alpha\",\n      \"connected\": true,\n      \"connects\": 1,\n      \"disconnects\": 0,\n      \"bytes_in\": 0,\n      \"bytes_out\": 0,\n      \"msg_in\": 0,\n      \"msg_out\": 0,\n      \"count\": 0,\n      \"rma\": 0,\n      \"q50\": -1,\n      \"q75\": -1,\n      \"q90\": -1,\n      \"q95\": -1\n    },\n    {\n      \"name\": \"Stan:bar to Stan:bar.out\",\n      \"id\": \"alpha\",\n      \"connected\": true,\n      \"connects\": 1,\n      \"disconnects\": 0,\n      \"bytes_in\": 0,\n      \"bytes_out\": 0,\n      \"msg_in\": 0,\n      \"msg_out\": 0,\n      \"count\": 0,\n      \"rma\": 0,\n      \"q50\": -1,\n      \"q75\": -1,\n      \"q90\": -1,\n      \"q95\": -1\n    }\n  ],\n  \"http_requests\": {\n    \"/\": 0,\n    \"/healthz\": 0,\n    \"/varz\": 799\n  }\n}"
```

Versus as is:

```
[12319] 2019/08/18 17:09:53.993542 [TRC] Retrieved metric result:
{
  "start_time": 1566152463,
  "current_time": 1566169793,
  "uptime": "4h48m49.918582365s",
  "request_count": 0,
  "connectors": [
    {
      "name": "Stan:foo to Stan:foo.out",
      "id": "alpha",
      "connected": true,
      "connects": 1,
      "disconnects": 0,
      "bytes_in": 0,
      "bytes_out": 0,
      "msg_in": 0,
      "msg_out": 0,
      "count": 0,
      "rma": 0,
      "q50": -1,
      "q75": -1,
      "q90": -1,
      "q95": -1
    },
    {
      "name": "Stan:bar to Stan:bar.out",
      "id": "alpha",
      "connected": true,
      "connects": 1,
      "disconnects": 0,
      "bytes_in": 0,
      "bytes_out": 0,
      "msg_in": 0,
      "msg_out": 0,
      "count": 0,
      "rma": 0,
      "q50": -1,
      "q75": -1,
      "q90": -1,
      "q95": -1
    }
  ],
  "http_requests": {
    "/": 0,
    "/healthz": 0,
    "/varz": 823
  }
}
```

wdyt?

Reply:
You are correct.